extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/HueConfigStatusMessage.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2014-2016 openHAB UG (haftungsbeschraenkt) and others.
```

Comment:
could you please update the license header to the [current one](https://github.com/eclipse/smarthome/blob/master/src/etc/header.txt)?